pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -171,14 +171,15 @@ func StartDNSProxy(address string, port uint16, lookupEPFunc LookupEndpointIDByI
 
 	// Start the DNS listeners on UDP and TCP
 	var (
-		UDPConn     *net.UDPConn
-		TCPListener *net.TCPListener
-		err         error
+		UDPConn                *net.UDPConn
+		TCPListener            *net.TCPListener
+		err                    error
+		EnableIPv4, EnableIPv6 = option.Config.EnableIPv4, option.Config.EnableIPv6
 	)
 
 	start := time.Now()
 	for time.Since(start) < ProxyBindTimeout {
-		UDPConn, TCPListener, err = bindToAddr(address, port)
+		UDPConn, TCPListener, err = bindToAddr(address, port, option.Config.EnableIPv4, option.Config.EnableIPv6)
```

Comment:
Why not pass `EnableIPv4` and `EnableIPv6` here ?

Reply:
Because I forgot! Good catch :)